<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}

.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Junior Research Group on Biomedical Big Data</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">News</a>
</li>
<li>
  <a href="people.html">Members</a>
</li>
<li>
  <a href="software.html">Software Tools</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="head.html">Head</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="contact.html">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/biomedbigdata">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/itisalist">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://de.linkedin.com/in/markus-list-3ba80927">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<div id="refs" class="references">
<div id="ref-Schmidt2018-fv">
<p>Schmidt, F., List, M., Cukuroglu, E., Köhler, S., Göke, J., &amp; Schulz, M. H. (2018). An ontology-based method for assessing batch effect adjustment approaches in heterogeneous datasets. <em>Bioinformatics</em>, <em>34</em>(17), i908–i916. <a href="https://doi.org/10.1093/bioinformatics/bty553" class="uri">https://doi.org/10.1093/bioinformatics/bty553</a></p>
</div>
<div id="ref-Hornakova2018-lz">
<p>Hornakova, A., List, M., Vreeken, J., &amp; Schulz, M. H. (2018). JAMI - fast computation of conditional mutual information for ceRNA network analysis. <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/bty221" class="uri">https://doi.org/10.1093/bioinformatics/bty221</a></p>
</div>
<div id="ref-Sippl2018-rr">
<p>Sippl, C., Ketter, R., Bohr, L., Kim, Y. J., List, M., Oertel, J., &amp; Urbschat, S. (2018). MiRNA-181d expression significantly affects treatment responses to carmustine wafer implantation. <em>Neurosurgery</em>. <a href="https://doi.org/10.1093/neuros/nyy214" class="uri">https://doi.org/10.1093/neuros/nyy214</a></p>
</div>
<div id="ref-Albrecht2017-oa">
<p>Albrecht, F., List, M., Bock, C., &amp; Lengauer, T. (2017). DeepBlueR: Large-scale epigenomic analysis in R. <em>Bioinformatics</em>, (February), 1–2. <a href="https://doi.org/10.1093/bioinformatics/btx099" class="uri">https://doi.org/10.1093/bioinformatics/btx099</a></p>
</div>
<div id="ref-Alcaraz2017-zg">
<p>Alcaraz, N., List, M., Batra, R., Vandin, F., Ditzel, H. J., &amp; Baumbach, J. (2017). De novo pathway-based biomarker identification. <em>Nucleic Acids Res.</em>, <em>45</em>(16), e151–e151. <a href="https://doi.org/10.1093/nar/gkx642" class="uri">https://doi.org/10.1093/nar/gkx642</a></p>
</div>
<div id="ref-Lund2017-ly">
<p>Lund, J. B., List, M., &amp; Baumbach, J. (2017). Interactive microbial distribution analysis using BioAtlas. <em>Nucleic Acids Res.</em>, <em>45</em>(W1), W509–W513. <a href="https://doi.org/10.1093/nar/gkx304" class="uri">https://doi.org/10.1093/nar/gkx304</a></p>
</div>
<div id="ref-Batra2017-dt">
<p>Batra, R., Alcaraz, N., Gitzhofer, K., Pauling, J., Ditzel, H. J., Hellmuth, M., … List, M. (2017). On the performance of de novo pathway enrichment. <em>Npj Systems Biology and Applications</em>, <em>3</em>(1), 6. <a href="https://doi.org/10.1038/s41540-017-0007-2" class="uri">https://doi.org/10.1038/s41540-017-0007-2</a></p>
</div>
<div id="ref-Elnegaard2017-hj">
<p>Elnegaard, M. P., List, M., Christiansen, H., Schmidt, S., Mollenhauer, J., &amp; Block, I. (2017). Protein-based nanotoxicology assessment strategy. <em>Nanomedicine</em>, <em>13</em>(3), 1229–1233. <a href="https://doi.org/10.1016/j.nano.2016.12.020" class="uri">https://doi.org/10.1016/j.nano.2016.12.020</a></p>
</div>
<div id="ref-Wittig-Blaich2017-ds">
<p>Wittig-Blaich, S., Wittig, R., Schmidt, S., Lyer, S., Bewerunge-Hudler, M., Gronert-Sum, S., … Mollenhauer, J. (2017). Systematic screening of isogenic cancer cells identifies DUSP6 as context-specific synthetic lethal target in melanoma. <em>Oncotarget</em>. <a href="https://doi.org/10.18632/oncotarget.15863" class="uri">https://doi.org/10.18632/oncotarget.15863</a></p>
</div>
<div id="ref-List2017-vg">
<p>List, M., Ebert, P., &amp; Albrecht, F. (2017). Ten simple rules for developing usable software in computational biology. <em>PLoS Comput. Biol.</em>, <em>13</em>(1), e1005265. <a href="https://doi.org/10.1371/journal.pcbi.1005265" class="uri">https://doi.org/10.1371/journal.pcbi.1005265</a></p>
</div>
<div id="ref-Schultheiss2017-ia">
<p>Schultheiss, C. S., Laggai, S., Czepukojc, B., Hussein, U. K., List, M., Barghash, A., … Kessler, S. M. (2017). The long non-coding RNA H19 suppresses carcinogenesis and chemoresistance in hepatocellular carcinoma. <em>CST</em>, <em>1</em>(1), 37–54. <a href="https://doi.org/10.15698/cst2017.10.105" class="uri">https://doi.org/10.15698/cst2017.10.105</a></p>
</div>
<div id="ref-List2017-ue">
<p>List, M. (2017). Using docker compose for the simple deployment of an integrated drug target screening platform. <em>J. Integr. Bioinform.</em>, <em>14</em>(2). <a href="https://doi.org/10.1515/jib-2017-0016" class="uri">https://doi.org/10.1515/jib-2017-0016</a></p>
</div>
<div id="ref-List2016-is">
<p>List, M., Schmidt, S., Christiansen, H., Rehmsmeier, M., Tan, Q., Mollenhauer, J., &amp; Baumbach, J. (2016). Comprehensive analysis of high-throughput screens with HiTSeekR. <em>Nucleic Acids Res.</em>, gkw554. <a href="https://doi.org/10.1093/nar/gkw554" class="uri">https://doi.org/10.1093/nar/gkw554</a></p>
</div>
<div id="ref-Albrecht2016-bm">
<p>Albrecht, F., List, M., Bock, C., &amp; Lengauer, T. (2016). DeepBlue epigenomic data server: Programmatic data retrieval and analysis of epigenome region sets. <em>Nucleic Acids Res.</em>, (i), gkw211. <a href="https://doi.org/10.1093/nar/gkw211" class="uri">https://doi.org/10.1093/nar/gkw211</a></p>
</div>
<div id="ref-List2016-kj">
<p>List, M., Elnegaard, M. P., Schmidt, S., Christiansen, H., Tan, Q., Mollenhauer, J., &amp; Baumbach, J. (2016). Efficient management of High-Throughput screening libraries with SAVANAH. <em>J. Biomol. Screen.</em>, 1087057116673607. <a href="https://doi.org/10.1177/1087057116673607" class="uri">https://doi.org/10.1177/1087057116673607</a></p>
</div>
<div id="ref-List2016-nj">
<p>List, M., Alcaraz, N., Dissing-Hansen, M., Ditzel, H. J., Mollenhauer, J., &amp; Baumbach, J. (2016). KeyPathwayMinerWeb: Online multi-omics network enrichment. <em>Nucleic Acids Res.</em>, gkw373. <a href="https://doi.org/10.1093/nar/gkw373" class="uri">https://doi.org/10.1093/nar/gkw373</a></p>
</div>
<div id="ref-Alcaraz2016-by">
<p>Alcaraz, N., List, M., Dissing-Hansen, M., Rehmsmeier, M., Tan, Q., Mollenhauer, J., … Baumbach, J. (2016). Robust de novo pathway enrichment with KeyPathwayMiner 5. <em>F1000Res.</em>, <em>5</em>(0), 1531. <a href="https://doi.org/10.12688/f1000research.9054.1" class="uri">https://doi.org/10.12688/f1000research.9054.1</a></p>
</div>
<div id="ref-List2015-pq">
<p>List, M., Franz, M., Tan, Q., Mollenhauer, J., &amp; Baumbach, J. (2015). OpenLabNotes - an electronic laboratory notebook extension for OpenLabFramework. <em>J. Integr. Bioinform.</em>, <em>12</em>(3), 274. <a href="https://doi.org/10.2390/biecoll-jib-2015-274" class="uri">https://doi.org/10.2390/biecoll-jib-2015-274</a></p>
</div>
<div id="ref-List2014-ae">
<p>List, M., Hauschild, A.-C., Tan, Q., Kruse, T. A., Mollenhauer, J., Baumbach, J., &amp; Batra, R. (2014a). Classification of breast cancer subtypes by combining gene expression and DNA methylation data. <em>J. Integr. Bioinform.</em>, <em>11</em>(2), 236. <a href="https://doi.org/10.2390/biecoll-jib-2014-236" class="uri">https://doi.org/10.2390/biecoll-jib-2014-236</a></p>
</div>
<div id="ref-List2014-ro">
<p>List, M., Hauschild, A.-C., Tan, Q., Kruse, T. A., Mollenhauer, J., Baumbach, J., &amp; Batra, R. (2014b). Classification of breast cancer subtypes by combining gene expression and DNA methylation data. <em>J. Integr. Bioinform.</em>, <em>11</em>(2), 236. <a href="https://doi.org/10.2390/biecoll-jib-2014-236" class="uri">https://doi.org/10.2390/biecoll-jib-2014-236</a></p>
</div>
<div id="ref-List2014-dd">
<p>List, M., Schmidt, S., Trojnar, J., Thomas, J., Thomassen, M., Kruse, T. A., … Mollenhauer, J. (2014). Efficient sample tracking with OpenLabFramework. <em>Sci. Rep.</em>, <em>4</em>. <a href="https://doi.org/10.1038/srep04278" class="uri">https://doi.org/10.1038/srep04278</a></p>
</div>
<div id="ref-List2014-qf">
<p>List, M., Block, I., Pedersen, M. L., Christiansen, H., Schmidt, S., Thomassen, M., … Mollenhauer, J. (2014). Microarray r-based analysis of complex lysate experiments with MIRACLE. <em>Bioinformatics</em>, <em>30</em>(17), i631–i638. <a href="https://doi.org/10.1093/bioinformatics/btu473" class="uri">https://doi.org/10.1093/bioinformatics/btu473</a></p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
