<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 2.4.0">
  <meta name="generator" content="Hugo 0.49" />
  <meta name="author" content="Markus List">

  
  
  
  
    
  
  <meta name="description" content="The composition and density of immune cells in the tumor microenvironment profoundly influence tumor progression and success of anti-cancer therapies. Flow cytometry, immunohistochemistry staining, or single-cell sequencing is usually not available such that we rely on computational methods to estimate the immune-cell composition from bulk RNA-sequencing (RNA-seq) data. Various methods have been proposed recently, yet their capabilities and limitations have not been evaluated systematically. A general guideline leading the research community through cell type deconvolution is missing. We developed a systematic approach for benchmarking such computational methods and assessed the accuracy of tools for estimating a variety of immune cell types from bulk RNA-seq samples. We used a single-cell RNA-seq dataset of ~11,000 cells from the tumor microenvironment to simulate bulk samples of known cell type proportions, which allowed us to measure the accuracy of the computational methods. Moreover, we collected publicly available data that provide both RNA-seq and independent, gold-standard estimates of the cell type proportions. This allowed us to analyze and condense the results of a hundred of thousand predictions to provide an exhaustive evaluation across computational methods over ten cell types and ~1,400 samples from five simulated and real-world datasets. We demonstrate that computational deconvolution performs at high accuracy for well-defined high-quality signatures, demonstrating its practical utility of bulk RNA-seq to dissect immune-cell composition of tumor samples. In addition to guidelines for method selection (depending on the cell types of interest) we also offer an R package to allow the community to perform integrated deconvolution using different methods themselves (https://grst.github.io/immunedeconv). We suggest that future efforts should be dedicated to refining cell population definitions and finding reliable signatures.">

  
  <link rel="alternate" hreflang="en-us" href="https://big-data-biomedicine.de/publication/cell_type_deconvolution/">

  


  

  
  
  
  <meta name="theme-color" content="hsl(36, 90%, 58%)">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/dracula.min.css" crossorigin="anonymous">
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.css" integrity="sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==" crossorigin="anonymous">
    

    

  

  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700%7cRoboto:400,400italic,700%7cRoboto&#43;Mono">
  

  <link rel="stylesheet" href="/styles.css">
  

  
  
  

  
  <link rel="alternate" href="https://big-data-biomedicine.de/index.xml" type="application/rss+xml" title="Big Data in Biomedicine">
  <link rel="feed" href="https://big-data-biomedicine.de/index.xml" type="application/rss+xml" title="Big Data in Biomedicine">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="https://big-data-biomedicine.de/publication/cell_type_deconvolution/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@itisalist">
  <meta property="twitter:creator" content="@itisalist">
  
  <meta property="og:site_name" content="Big Data in Biomedicine">
  <meta property="og:url" content="https://big-data-biomedicine.de/publication/cell_type_deconvolution/">
  <meta property="og:title" content="Comprehensive evaluation of cell-type quantification methods for immuno-oncology | Big Data in Biomedicine">
  <meta property="og:description" content="The composition and density of immune cells in the tumor microenvironment profoundly influence tumor progression and success of anti-cancer therapies. Flow cytometry, immunohistochemistry staining, or single-cell sequencing is usually not available such that we rely on computational methods to estimate the immune-cell composition from bulk RNA-sequencing (RNA-seq) data. Various methods have been proposed recently, yet their capabilities and limitations have not been evaluated systematically. A general guideline leading the research community through cell type deconvolution is missing. We developed a systematic approach for benchmarking such computational methods and assessed the accuracy of tools for estimating a variety of immune cell types from bulk RNA-seq samples. We used a single-cell RNA-seq dataset of ~11,000 cells from the tumor microenvironment to simulate bulk samples of known cell type proportions, which allowed us to measure the accuracy of the computational methods. Moreover, we collected publicly available data that provide both RNA-seq and independent, gold-standard estimates of the cell type proportions. This allowed us to analyze and condense the results of a hundred of thousand predictions to provide an exhaustive evaluation across computational methods over ten cell types and ~1,400 samples from five simulated and real-world datasets. We demonstrate that computational deconvolution performs at high accuracy for well-defined high-quality signatures, demonstrating its practical utility of bulk RNA-seq to dissect immune-cell composition of tumor samples. In addition to guidelines for method selection (depending on the cell types of interest) we also offer an R package to allow the community to perform integrated deconvolution using different methods themselves (https://grst.github.io/immunedeconv). We suggest that future efforts should be dedicated to refining cell population definitions and finding reliable signatures."><meta property="og:image" content="https://big-data-biomedicine.de/img/immunedeconv.png">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-11-07T00:00:00&#43;01:00">
  
  <meta property="article:modified_time" content="2018-11-07T00:00:00&#43;01:00">
  

  

  

  <title>Comprehensive evaluation of cell-type quantification methods for immuno-oncology | Big Data in Biomedicine</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" class="dark">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">Big Data in Biomedicine</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        

        
        
        
          
        

        <li class="nav-item">
          <a href="/#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a href="/group">
            
            <span>Group</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a href="/#publications_selected">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a href="/#posts">
            
            <span>Posts</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a href="/#projects">
            
            <span>Projects</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a href="/#teaching">
            
            <span>Teaching</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>

<div class="pub" itemscope itemtype="http://schema.org/CreativeWork">

  
<div class="article-header">
  
  
    <img src="/img/immunedeconv.png" class="article-banner" itemprop="image">
  

  <span class="article-header-caption">Spillover analysis in which all methods were applied to 30 simulated bulk RNA-seq samples of six immune cell types. The plots show the ratio of correct vs. false prediction and unveil colinearities between the cell types. The numbers indicate the overall noise ratio, i.e. the fraction of predictions that are attributed to a wrong cell type.</span>
</div>



  <div class="article-container">
    <h1 itemprop="name">Comprehensive evaluation of cell-type quantification methods for immuno-oncology</h1>
    <span class="pub-authors" itemprop="author">
      
      Gregor Sturm, Francesca Finotello, Florent Petitprez, Jitao David Zheng, Jan Baumbach, Wolf H Fridman, Markus List, Tasiana Aneichyk
      
    </span>
    <span class="pull-right">
      
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Comprehensive%20evaluation%20of%20cell-type%20quantification%20methods%20for%20immuno-oncology&amp;url=https%3a%2f%2fbig-data-biomedicine.de%2fpublication%2fcell_type_deconvolution%2f"
         target="_blank" rel="noopener">
        <i class="fab fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=https%3a%2f%2fbig-data-biomedicine.de%2fpublication%2fcell_type_deconvolution%2f"
         target="_blank" rel="noopener">
        <i class="fab fa-facebook-f"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=https%3a%2f%2fbig-data-biomedicine.de%2fpublication%2fcell_type_deconvolution%2f&amp;title=Comprehensive%20evaluation%20of%20cell-type%20quantification%20methods%20for%20immuno-oncology"
         target="_blank" rel="noopener">
        <i class="fab fa-linkedin-in"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=https%3a%2f%2fbig-data-biomedicine.de%2fpublication%2fcell_type_deconvolution%2f&amp;title=Comprehensive%20evaluation%20of%20cell-type%20quantification%20methods%20for%20immuno-oncology"
         target="_blank" rel="noopener">
        <i class="fab fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Comprehensive%20evaluation%20of%20cell-type%20quantification%20methods%20for%20immuno-oncology&amp;body=https%3a%2f%2fbig-data-biomedicine.de%2fpublication%2fcell_type_deconvolution%2f">
        <i class="fas fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


    </span>

    

    
    <h3>Abstract</h3>
    <p class="pub-abstract" itemprop="text">The composition and density of immune cells in the tumor microenvironment profoundly influence tumor progression and success of anti-cancer therapies. Flow cytometry, immunohistochemistry staining, or single-cell sequencing is usually not available such that we rely on computational methods to estimate the immune-cell composition from bulk RNA-sequencing (RNA-seq) data. Various methods have been proposed recently, yet their capabilities and limitations have not been evaluated systematically. A general guideline leading the research community through cell type deconvolution is missing. We developed a systematic approach for benchmarking such computational methods and assessed the accuracy of tools for estimating a variety of immune cell types from bulk RNA-seq samples. We used a single-cell RNA-seq dataset of ~11,000 cells from the tumor microenvironment to simulate bulk samples of known cell type proportions, which allowed us to measure the accuracy of the computational methods. Moreover, we collected publicly available data that provide both RNA-seq and independent, gold-standard estimates of the cell type proportions. This allowed us to analyze and condense the results of a hundred of thousand predictions to provide an exhaustive evaluation across computational methods over ten cell types and ~1,400 samples from five simulated and real-world datasets. We demonstrate that computational deconvolution performs at high accuracy for well-defined high-quality signatures, demonstrating its practical utility of bulk RNA-seq to dissect immune-cell composition of tumor samples. In addition to guidelines for method selection (depending on the cell types of interest) we also offer an R package to allow the community to perform integrated deconvolution using different methods themselves (<a href="https://grst.github.io/immunedeconv" target="_blank">https://grst.github.io/immunedeconv</a>). We suggest that future efforts should be dedicated to refining cell population definitions and finding reliable signatures.</p>
    

    
    <div class="row">
      <div class="col-sm-1"></div>
      <div class="col-sm-10">
        <div class="row">
          <div class="col-xs-12 col-sm-3 pub-row-heading">Type</div>
          <div class="col-xs-12 col-sm-9">
            
            <a href="https://big-data-biomedicine.de/publication/#7">
              Preprint
            </a>
            
          </div>
        </div>
      </div>
      <div class="col-sm-1"></div>
    </div>
    <div class="visible-xs space-below"></div>
    

    
    <div class="row">
      <div class="col-sm-1"></div>
      <div class="col-sm-10">
        <div class="row">
          <div class="col-xs-12 col-sm-3 pub-row-heading">Publication</div>
          <div class="col-xs-12 col-sm-9">Comprehensive evaluation of cell-type quantification methods for immuno-oncology</div>
        </div>
      </div>
      <div class="col-sm-1"></div>
    </div>
    <div class="visible-xs space-below"></div>
    

    <div class="row">
      <div class="col-sm-1"></div>
      <div class="col-sm-10">
        <div class="row">
          <div class="col-xs-12 col-sm-3 pub-row-heading">Date</div>
          <div class="col-xs-12 col-sm-9" itemprop="datePublished">
            November, 2018
          </div>
        </div>
      </div>
      <div class="col-sm-1"></div>
    </div>
    <div class="visible-xs space-below"></div>

    <div class="row" style="padding-top: 10px">
      <div class="col-sm-1"></div>
      <div class="col-sm-10">
        <div class="row">
          <div class="col-xs-12 col-sm-3 pub-row-heading" style="line-height:34px;">Links</div>
          <div class="col-xs-12 col-sm-9">

            




<a class="btn btn-primary btn-outline" href="https://www.biorxiv.org/content/early/2018/11/14/463828.full.pdf" target="_blank" rel="noopener">
  PDF
</a>





<a class="btn btn-primary btn-outline" href="https://github.com/grst/immunedeconv" target="_blank" rel="noopener">
  Code
</a>












          </div>
        </div>
      </div>
      <div class="col-sm-1"></div>
    </div>
    <div class="visible-xs space-below"></div>

    <div class="space-below"></div>

    <div class="article-style"></div>

    


<div class="article-tags">
  
  <a class="label label-default" href="https://big-data-biomedicine.de/tags/single-cell/">single-cell</a>
  
  <a class="label label-default" href="https://big-data-biomedicine.de/tags/cell-type-deconvolution/">cell-type deconvolution</a>
  
  <a class="label label-default" href="https://big-data-biomedicine.de/tags/tumor-microenvironment/">tumor microenvironment</a>
  
  <a class="label label-default" href="https://big-data-biomedicine.de/tags/cancer-immunology/">cancer immunology</a>
  
</div>




  </div>
</div>



<footer class="site-footer">
  <div class="container">

    

    <p class="powered-by">

      &copy; 2018 &middot; 

      Powered by the
      <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
      <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fas fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
        
      

      
      
    

    <script src="/js/hugo-academic.js"></script>
    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.js" integrity="sha512-lInM/apFSqyy1o6s89K4iQUKg6ppXEgsVxT35HbzUupEVRh2Eu9Wdl4tHj7dZO0s1uvplcYGmt3498TtHq+log==" crossorigin="anonymous"></script>
    

    
    
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    <script src="/js/search.js"></script>
    

    
    

  </body>
</html>

